src/main/java/picard/cmdline/PicardCommandLine.java

```diff
@@ -278,6 +292,20 @@ private static void printUsage(final Set<Class<?>> classes, final String command
         }
     }
 
+    public static String getFeaturePrefix(Class clazz){
```

Comment:
I don't think this needs to be public (though if it does, it should have javadoc). It might be better named something like getToolSummaryPrefix. Also, even though the code in this file is inconsistent about use of raw types, it would be nice to avoid introducing new methods that use them (i.e., Class as opposed to Class<?>).

Reply:
@cmnbroad , thank's for review, I've done all requested changes, excludes highlighting prefixes in red, may be it would be better to use another(not red) color for it experimental/beta feature?